tests/unit/sagemaker/jumpstart/model/test_model.py

```diff
@@ -170,6 +170,43 @@ def test_prepacked(
             ],
         )
 
+    @mock.patch("sagemaker.model.LOGGER.warning")
+    @mock.patch("sagemaker.utils.sagemaker_timestamp")
+    @mock.patch("sagemaker.session.Session.endpoint_from_production_variants")
+    @mock.patch("sagemaker.session.Session.create_model")
```

Comment:
nit: Can we mock these functions from the `mock_session` patch?

Reply:
yeah let me see if i can simplify this, it was lifted from another test

Reply:
this is a different mocked session object we need to include. It's messy i know lol